Source/JavaScriptCore/dfg/DFGCriticalEdgeBreakingPhase.cpp

```diff
@@ -46,6 +46,8 @@ class CriticalEdgeBreakingPhase : public Phase {
     
     bool run()
     {
+        m_graph.m_newJumpPads.clear();
+
```

Comment:
Ah, ugh, yeah, this looks a bit weird. So how about doing this?
We first collect newly inserted jump pads here.
```
Vector<BasicBlock*> newJumpPads;
```